fdbserver/VersionedBTree.actor.cpp

```diff
@@ -1894,6 +1894,12 @@ class DWALPager : public IPager2 {
 			self->remapQueue.resetHeadReader();
 
 			self->remapCleanupFuture = remapCleanup(self);
+			TraceEvent(SevInfo, "RedwoodRecovered")
+			    .detail("FilePrefix", self->filename.c_str())
```

Comment:
Is this a file prefix at this point or a full FileName, with the `.redwood` extension?  I thought it was the latter.

Reply:
You are right. Confirmed that it's full FileName. Fixed.